proofs/sectorbuilder/interface.go

```diff
@@ -90,12 +89,11 @@ type SealedSectorMetadata struct {
 
 // GeneratePoStRequest represents a request to generate a proof-of-spacetime.
 type GeneratePoStRequest struct {
-	SortedCommRs  proofs.SortedCommRs
-	ChallengeSeed types.PoStChallengeSeed
+	SortedSectorInfo go_sectorbuilder.SortedSectorInfo
+	ChallengeSeed    types.PoStChallengeSeed
 }
 
 // GeneratePoStResponse contains PoST proof and any faults that may have occurred.
```

Comment:
Update this comment, as my understanding is the faults aren't coming back here.

Reply:
done!